spec/models/project_mediaflux_spec.rb

```diff
@@ -98,7 +98,7 @@
           collection_id = ProjectMediaflux.create!(project: incomplete_project, session_id: session_token)
         }.to raise_error do |error|
           expect(error).to be_a(TigerData::MetadataError)
-          expect(error.message).to include("Project failed to create with metadata schema version 0.6 with the missing fields:")
+          expect(error.message).to include("Project creation failed with metadata schema version 0.6 due to the missing fields:")
```

Comment:
Could we add which fields are missing? That seems like an important bit of functionality that we want to ensure we're testing.

Reply:
Yes we can! I'll work on that now